### ISSUE TITLE
New / improved accessibility themes

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1366]
Accessibility options should include options to select alternative themes to meet user's requirements. Some of these already exist (but could do with tweaks to improve them) and others need to be created from scratch:

- [x] dark mode (new)
- [x] light mode (new)
- [x] high contrast
- [x] black on yellow
